DependencyInjection/FOSRestExtension.php

```diff
@@ -230,6 +231,12 @@ private function loadVersioning(array $config, XmlFileLoader $loader, ContainerB
     private function loadParamFetcherListener(array $config, XmlFileLoader $loader, ContainerBuilder $container)
     {
         if ($config['param_fetcher_listener']['enabled']) {
+            if (!class_exists(Constraint::class)) {
+                throw new \RuntimeException(
```

Comment:
I am afraid I was too eager by suggesting to throw an exception here. There may be applications where someone enabled the option without using the feature at all. Having these applications break on an update is not the best experience. I think we should instead just trigger a deprecation here.

Reply:
Indeed it might be more prudent. But the developper might miss it too because it's only a deprecation. The initial goal of the PR is too avoid a "class not found" message. What do you think about doing both things:
- Add a deprecation at compile time
- Fix the validator check position like I suggest it earlier ([69e1d7](https://github.com/FriendsOfSymfony/FOSRestBundle/commit/69e1d7f94e1ff30a7dcd1b680ea3cdbecd5745a3)) to have a more mainingfull message than "class not found" ?

Reply:
I pushed the suggested changes